### ISSUE TITLE
5751 N. Central Ave: 19-164493

### ISSUE THREAD

**Issue by Mtierneyo** [meridian-kessler/Code-Compliance#7]

**Comment by cmarburger08** [comment]
construction underway but starts and stops for months at a time; north setback appears insufficient.  Reported to MAC on 9/30/19 - MO

**Comment by cmarburger08** [comment]
MAC Update: Zoning violation issues on 9/26/19 - no further details available.

**Comment by Mtierneyo** [comment]
Not sure what this means...Is the case "closed"?


-----Original Message-----

**Comment by cmarburger08** [comment]
No. The ticket will be closed once the case is closed. This particular property has been reported more than once and a violation on the 9/26/19, four days before we sent in our ticket. There is a zoning violation on the property right now. Those were the only details that I could get over the phone today.

**Comment by cmarburger08** [comment]
@JDKingen to pull site plan for STR17-05146/ILP17-02306 - will follow up next week. 

INV19-36370 investigation issued on 9/30/19. Investigation turned up no violations: 
<img width="818" alt="Screen Shot 2019-10-25 at 2 28 57 PM" src="https://user-images.githubusercontent.com/55200090/67595099-d3c7f300-f733-11e9-98c5-5ddf55a9d835.png">
